diagnostics/include/diagnostics_server.h

```diff
@@ -90,6 +90,7 @@ inline std::string readline(int sock) {
 
 inline void handleRequest(Registrar& registrar, int sock) {
   try {
+    LOG_INFO(logger, "Handle Diagnostics Request");
```

Comment:
Hi Andrew, AFAI, the diagnostics server is called by the agent every 30 seconds, so I'm afraid this might pollute the logs.
Can we make this a DEBUG log?